designcompose/src/main/java/com/android/designcompose/Utils.kt

```diff
@@ -1245,6 +1245,22 @@ internal fun StrokeWeight.right(): Float {
 internal fun ViewStyle.isAutoWidthText() =
     width is Dimension.Auto && horizontal_sizing !is LayoutSizing.FILL
 
+internal fun getNodeRenderSize(
```

Comment:
Can you add a comment documenting the different scenarios here? I wasn't sure what all the cases were that `getRect` was handling previously were, and it took me a while to find the regressions when I changed it. (I was animating the layout size, and was surprised when some elements didn't animate).

Reply:
Done